experiments/chat/chat/templates/chat/room_detail.html

```diff
@@ -13,16 +13,15 @@
 </head>
 <body>
 <h1>{{ room.name }}</h1>
-<turbo-channels-stream-source model="chat.Message" stream="room_id" value="{{ room.id }}">
-</turbo-channels-stream-source>
-<turbo-frame id="messages-frame">
-     <ul id="messages">
-        {% for message in room.messages.all %}
-            {% include "chat/message.html" with message=message only %}
-        {% endfor %}
-    </ul>
-</turbo-frame>
-<turbo-frame id="send-message" src="{% url 'send' room.id %}">
-</turbo-frame>
+<turbo-channels-stream-source model="chat.Message" stream="room_id" value="{{ room.id }}" />
+
+<ul id="messages">
+  {% for message in room.messages.all %}
+      {% include "chat/message.html" with message=message only %}
+  {% endfor %}
+</ul>
+
+{% include "chat/message_form_frame.html" %}
```

Comment:
This definitely works, and is probably a cleaner to include the form on the back-end based on how the template partials are split up, but it's important to note that including the message form this way means we're not fetching the `send-message` frame on the client -- the old code used the `src` attribute on the frame to fetch the frame from the `/send` page. Either way is fine, and honestly hotwire is so new there probably isn't a "right" way to do this right now, but should probably add a comment here just to make it clear